File: https:/github.com/sajith48/leetcode_solutions.git/1476-CountNegativeNumbersInASortedMatrix/1476-CountNegativeNumbersInASortedMatrix.c
// Last updated: 7/18/2026, 9:26:10 PM
int countNegatives(int** grid, int gridSize, int* gridColSize) {
    int c=0;
    for(int i=0;i<gridSize;i++){
        for(int j=0;j<*gridColSize;j++){
            if(grid[i][j]<0)c++;
        }
    }
    return c;
}

File: https:/github.com/sajith48/leetcode_solutions.git/1476-CountNegativeNumbersInASortedMatrix/1476-CountNegativeNumbersInASortedMatrix.java
// Last updated: 7/18/2026, 9:26:04 PM
class Solution {
    public int countNegatives(int[][] grid) {
        int neg=0;
        for(int i=0;i<grid.length;i++){
            for(int j=0;j<grid[i].length;j++){
                if(grid[i][j]<0)neg+=1;
            }
        }
        return neg;
    }
}